server/src/services/cleanUserInput.js

```diff
@@ -0,0 +1,10 @@
+const cleanUserInput = formInput => {
+  Object.keys(formInput).forEach(field => {
+    if(formInput[field] === "") {
+      delete formInput[field]
+    }
+  })
+  return formInput
+}
```

Comment:
This file is not being used by this feature! We shouldn't over-engineer features/ functions/ things we think we might need in the future